### PR TITLE
Fix #155

### LINKED ISSUE #155: Storage button in Background Image settings is oversized
Screenshot: http://goo.gl/kjgmW4

Happens in both the presentation properties modal and the sidebar

Adding `btn-widget-icon-storage` and removing the `img` child seems to set it right, but needs testing etc.

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. Cheers

@rodrigopavezi FYI

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc the File selector on the Presentation Properties no longer works in this PR. Can you double check? Thanks!

File: index.html [hunk 1]
```diff
@@ -88,6 +88,7 @@
     <script src="bower_components/widget-settings-ui-components/dist/js/angular/tooltip.js"></script>
     <script src="bower_components/widget-settings-ui-components/dist/js/angular/url-field.js"></script>
     <script src="bower_components/widget-settings-ui-components/dist/js/angular/position-setting.js"></script>
+    <script src="bower_components/widget-settings-ui-components/dist/js/angular/file-selector.js"></script>
```
Comment (rodrigopavezi) [inline index.html]: @ezequielc you shouldn't need to load file-selector.

File: bower.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "rv-loading": "https://github.com/Rise-Vision/rv-loading.git",
     "rv-common-app-components": "https://github.com/Rise-Vision/common-app-components.git",
     "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git",
-    "component-storage-selector": "https://github.com/Rise-Vision/component-storage-selector.git",
+    "component-storage-selector": "https://github.com/Rise-Vision/component-storage-selector.git#1.0.1",
     "widget-settings-ui-components": "https://github.com/Rise-Vision/widget-settings-ui-components.git",
     "widget-settings-ui-core": "https://github.com/Rise-Vision/widget-settings-ui-core.git",
     "angular-bootstrap-colorpicker": "~3.0.19",
```
Comment (rodrigopavezi) [inline bower.json]: @ezequielc I think you are missing the resolution here. cheers

**Comment by ezequielc** [comment]
@rodrigopavezi The issue was that I was using a wrong version of widget-settings-ui-components. It should be working now. Could you review? Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good :+1: . Thanks.

**Pull request merged**